src/rpc/rawtransaction.cpp

```diff
@@ -338,6 +338,9 @@ static UniValue gettxchainlocks(const JSONRPCRequest& request)
         if (height != -1) {
             chainLock = llmq_ctx.clhandler->HasChainLock(height, hash_block);
         }
+        else {
+            result.pushKV("mempool", true);
```

Comment:
why should this not simply be; if GetTransaction finds it; then true, otherwise false?